src/_InputLabel.tsx

```diff
@@ -3,25 +3,41 @@ import {Box} from '.'
 import {SxProp} from './sx'
 import VisuallyHidden from './_VisuallyHidden'
 
-interface Props extends React.HTMLProps<HTMLLabelElement> {
+type BaseProps = SxProp & {
```

Comment:
(sorry, typescript noob here): Would removing `HTMLProps` throw warning for other html attributes like form or className?

Reply:
It would, but those props weren't getting forwarded to the underlying element anyway - so if you passed `className` you'd expect it to work but it wouldn't have actually worked. In that sense, this type was actually misleading. 

In addition, this is a private (internal) component and there wasn't anywhere we were using those types of props internally, so this shouldn't have any effect.